public/app/features/logs/components/panel/LogLineDetails.tsx

```diff
@@ -107,12 +62,9 @@ const getStyles = (theme: GrafanaTheme2) => ({
   }),
```

Comment:
A bit more differentiation between the details and the logs view with more consistent border/spacing makes it look a lot cleaner.

```suggestion
    boxShadow: theme.shadows.z2,
    border: `1px solid ${theme.colors.border.weak}`,
    borderRight: 'none',
    paddingTop: theme.spacing(0.5)
  }),
```

Reply:
Looks good! Probably even better without permanently visible scroll, like I have.